pyproject.toml

```diff
@@ -32,6 +32,7 @@ requires-python = ">=3.9"
 dependencies = [
   "click",
   "cookiecutter",
+  "tomlkit",
```

Comment:
TOML Kit is a zero-dependency TOML-writing library published by the Poetry organization. Having a structured and less hacky way of adding chosen dependencies `pyproject.toml` file feels worth it to me.